fail2ban/client/configparserinc.py

```diff
@@ -200,6 +200,18 @@ def get_defaults(self):
 	def get_sections(self):
 		return self._sections
 
+	def options(self, section, onlyOwn=False):
+		"""Return a list of option names for the given section name."""
```

Comment:
it might be obvious to you (and today) but it was not obvious to me on what kind of 'ownership' we are talking about ... also "only own" sounds awkward in english afaik... "only owned" would have been better but anyways imho it is not appropriate (once again imho) since ownership is a vague non-described idea here